README.rst

```diff
@@ -51,21 +51,21 @@ environment and added to the configuration template::
 
   dm-deploy bootstrap --proxy-env='FOO,BAR'
 
-Ephemeral environments
-~~~~~~~~~~~~~~~~~~~~~~
+Development environments
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create an ephemeral environment for a feature branch use the 
-``deploy-to-branch-environment`` command. This will create a development version,
-a development environment with that version and an associated RDS instance. If
-a branch is not explicitly provided then the current branch name will be used::
+To create a development environment (for a feature branch, for example) use the
+``deploy-to-development-environment`` command. This will create a development
+version, a development environment with that version and an associated RDS
+instance. A name for the development environment must be provided as the first
+argument::
```

Comment:
Should we specify the name of the app being deployed as part of the name for the development environment?  Now we have (or will have) several different applications it would be useful to always have the application name as part of the environment (e.g. 'api', 'search-api', 'admin-app', etc.)